app/controllers/meetings/publish/uittreksels.js

```diff
@@ -0,0 +1,74 @@
+import Controller from '@ember/controller';
+import {inject as service} from "@ember/service";
+import {task} from "ember-concurrency-decorators";
+import {tracked} from "@glimmer/tracking";
+
+export default class MeetingsPublishUittrekselsController extends Controller {
+  @service ajax;
+
+  @tracked
+  uittreksels = [];
+
+  constructor() {
+    super(...arguments);
+  }
+
+  initialize() {
+    this.initializeUittreksels.perform();
+  }
+
+
+  @task
+  * initializeUittreksels() {
+    const uittreksels = [];
+    const prePublish = yield this.createPrePublishedResource.perform();
+    for(const uittreksel of prePublish) {
```

Comment:
we could maybe extract this in a separate task and then run them concurrently with all() from ember-concurrency